src/views/tickets/rsvp.php

```diff
@@ -28,7 +28,7 @@ class="tribe-tickets-rsvp cart <?php echo esc_attr( $messages_class ); ?>"
 	enctype='multipart/form-data'
 >
 	<h2 class="tribe-events-tickets-title tribe--rsvp">
-		<?php echo esc_html_x( 'RSVP', 'form heading', 'event-tickets' ) ?>
+		<?php echo tribe_get_rsvp_label_singular( 'form_heading' ) ?>
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 4946be7c70fc140864af54379e16684a</sub>